app/src/main/java/com/example/ithaca_transit_android_v2/states/SearchState.kt

```diff
@@ -0,0 +1,78 @@
+package com.example.ithaca_transit_android_v2.states
+
+import com.example.ithaca_transit_android_v2.models.Coordinate
+import com.example.ithaca_transit_android_v2.models.Location
+
+//Parent Search State Class - Contains Default parameters
+sealed class SearchState(
+    val startLocation: Location = Location("", Coordinate(0.0,0.0),""),
+    val endLocation: Location = Location("", Coordinate(0.0,0.0),"") ,
+    val favDestinations: List<Location> = mutableListOf(),
+    val recentDestinations: List<Location> = mutableListOf()
```

Comment:
i'm trying to decide if we should leave these two fields out for now since it's outside the scope of v.0 - what do you think?

Reply:
Yea, I can leave them out for now. If we aren't implementing them in v.0, then they won't be necessary.